web/src/people.ts

```diff
@@ -118,6 +118,45 @@ export function init(): void {
 // WE INITIALIZE DATA STRUCTURES HERE!
 init();
 
+// A function that sorts Email according to the user's full name
+export function sort_emails(emails: string[], user_ids: number[]): string[] {
+    const namesArray = user_ids.map((user_id) => {
+        let person;
+        if (!people_by_user_id_dict.has(user_id)) {
+            person = undefined;
+        } else {
+            person = people_by_user_id_dict.get(user_id);
+        }
+        if (!person) {
+            return "?";
+        }
+        return person.full_name;
+    });
+    // Combine the non-undefined elements
+    const combinedArray = namesArray
+        .map((name, index) => ({
+            name,
+            email: emails[index],
+        }))
+        .filter((item) => item.name !== undefined);
+    // Sort the combined array based on the 'name' property.
+    combinedArray.sort((a, b) => {
+        const nameA = a.name.toLowerCase(); // Use ! to assert that a.name is not undefined
+        const nameB = b.name.toLowerCase(); // Use ! to assert that b.name is not undefined
+        return nameA.localeCompare(nameB);
+    });
+    const sortedemails = combinedArray.map((item) => item.email);
+    return sortedemails;
+}
+
+export function sort_names(names: string[]): string[] {
+    return names.sort((a, b) => {
+        const nameA = a.toLowerCase(); // Use ! to assert that a.name is not undefined
+        const nameB = b.toLowerCase(); // Use ! to assert that b.name is not undefined
+        return nameA.localeCompare(nameB);
+    });
```

Comment:
What is the need for this? You can just use `util.strcmp`.

Reply:
Okay, I will modify my code.